<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>759</width>
    <height>538</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>O Mundo Mágico de José</string>
  </property>
  <property name="windowIcon">
   <iconset resource="recursos.qrc">
    <normaloff>:/Tiles/img/Tiles/J.jpg</normaloff>:/Tiles/img/Tiles/J.jpg</iconset>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QFrame" name="frame_escolha_pers">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>110</y>
      <width>351</width>
      <height>151</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QPushButton" name="jogar">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>120</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Jogar</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>300</y>
       <width>191</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Selecione um personagem para jogar!!</string>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBox">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>80</y>
       <width>91</width>
       <height>21</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>Guerreiro</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Arqueiro</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Assassino</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Barbaro</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Mago</string>
      </property>
     </item>
    </widget>
    <widget class="QLabel" name="TextoPersonagem">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>141</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Escolha seu personagem</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_NomePers">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>80</y>
       <width>113</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="TextoNomePersonagem">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>50</y>
       <width>141</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Nome do Personagem</string>
     </property>
    </widget>
    <widget class="QPushButton" name="somAtivado">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>120</y>
       <width>28</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="recursos.qrc">
       <normaloff>:/Audio/img/Audio/SomLigado.png</normaloff>:/Audio/img/Audio/SomLigado.png</iconset>
     </property>
    </widget>
    <widget class="QPushButton" name="somDesativado">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>120</y>
       <width>28</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="recursos.qrc">
       <normaloff>:/Audio/img/Audio/SomDesligado.png</normaloff>:/Audio/img/Audio/SomDesligado.png</iconset>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="frame_novo_jogo">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>160</y>
      <width>511</width>
      <height>191</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>40</y>
       <width>361</width>
       <height>121</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="OpcaoInicial">
      <item>
       <widget class="QPushButton" name="novoJogo">
        <property name="text">
         <string>Novo Jogo</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="sairJogo">
        <property name="text">
         <string>Sair</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QFrame" name="statsPer">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>331</width>
      <height>111</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>132</width>
       <height>54</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <property name="sizeConstraint">
       <enum>QLayout::SetDefaultConstraint</enum>
      </property>
      <item>
       <layout class="QHBoxLayout" name="HP_layout">
        <item>
         <widget class="QLabel" name="HPTXT">
          <property name="text">
           <string>HP</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QProgressBar" name="HP_progbar">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="autoFillBackground">
           <bool>false</bool>
          </property>
          <property name="styleSheet">
           <string notr="true">color:rgb(0, 255, 0)</string>
          </property>
          <property name="value">
           <number>24</number>
          </property>
          <property name="textVisible">
           <bool>false</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="MP_layout">
        <item>
         <widget class="QLabel" name="MPTXT">
          <property name="text">
           <string>MP</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QProgressBar" name="MP_progbar">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="styleSheet">
           <string notr="true">color:rgb(0, 0, 255)</string>
          </property>
          <property name="value">
           <number>24</number>
          </property>
          <property name="textVisible">
           <bool>false</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
    <widget class="QLabel" name="label_nomePer">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>261</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>20</pointsize>
       <weight>75</weight>
       <italic>true</italic>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>NomePersonagem</string>
     </property>
    </widget>
    <widget class="QFrame" name="frame">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>120</width>
       <height>80</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="frame_game_over">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>120</y>
      <width>750</width>
      <height>200</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>10</y>
       <width>630</width>
       <height>195</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <property name="spacing">
       <number>22</number>
      </property>
      <item>
       <widget class="QLabel" name="label">
        <property name="font">
         <font>
          <pointsize>72</pointsize>
          <weight>75</weight>
          <italic>true</italic>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>GAME OVER!</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="encerraJogo">
        <property name="text">
         <string>Ok!</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QFrame" name="frame_vitoria">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>120</y>
      <width>750</width>
      <height>200</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QWidget" name="layoutWidget_3">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>10</y>
       <width>702</width>
       <height>163</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <property name="spacing">
       <number>22</number>
      </property>
      <item>
       <widget class="QLabel" name="label_3">
        <property name="font">
         <font>
          <pointsize>72</pointsize>
          <weight>75</weight>
          <italic>true</italic>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>VOCÊ VENCEU</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="encerraJogo_3">
        <property name="text">
         <string>Ok!</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <zorder>frame_novo_jogo</zorder>
   <zorder>statsPer</zorder>
   <zorder>frame_game_over</zorder>
   <zorder>frame_vitoria</zorder>
   <zorder>frame_escolha_pers</zorder>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="recursos.qrc"/>
 </resources>
 <connections/>
</ui>
